Tests/RxSwiftTests/PrimitiveSequenceTest.swift

```diff
@@ -1046,6 +1046,131 @@ extension PrimitiveSequenceTest {
 }
 
 extension PrimitiveSequenceTest {
+    func testFirst_Empty() {
+        let scheduler = TestScheduler(initialClock: 0)
+
+        let xs = scheduler.createHotObservable([
+            next(150, 1),
+            completed(250),
+            error(260, testError)
+            ])
+
+        let res: TestableObserver<Int> = scheduler.start { () -> Observable<Int> in
+            let single: Single<Int> = xs.first().map { $0 ?? -1 }
```

Comment:
I've tried to use optionals, but i get this error `Value of optional type '[Recorded<Event<Int?>>]' not unwrapped; did you mean to use '!' or '?'?`